plugins/become/sudosu.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    become: sudosu
+    short_description: Run tashks using sudo su -
+    description:
+        - This become plugins allows your remote/login user to execute commands as another user via the sudo and su utility combined.
+    author: ansible (@core)
+    version_added: "2.12"
+    options:
+        become_user:
+            description: User you 'become' to execute the task
```

Comment:
```suggestion
            description: User you 'become' to execute the task.
```

Reply:
This is coming from the original sudo plugin, but no issue in adding trailing dots.

Reply:
There are some plugins/modules in core that someone needs to go through as well, to adjust to all current docs guidelines...